server/src/domain/domain.constant.ts

```diff
@@ -27,3 +27,59 @@ export function assertMachineLearningEnabled() {
     throw new BadRequestException('Machine learning is not enabled.');
   }
 }
+
+const validMimeTypes = [
```

Comment:
This should be updated to match what's in main.

![image](https://github.com/immich-app/immich/assets/9749173/35347c82-4501-4cd8-80e9-2b5544f4cfd3)

Reply:
reverted this file to match in `main` = remove changes in this file